test/tech/v3/datatype/mmap_list_test.clj

```diff
@@ -0,0 +1,99 @@
+(ns tech.v3.datatype.mmap-list-test
+  (:require [tech.v3.datatype.mmap :as sut]
+            [clojure.test :as t]
+            [tech.v3.datatype.mmap :as mmap]
+            [clojure.java.io :as io]
+            [tech.v3.datatype :as dtype]
+            )
+
+(:import
+           [tech.v3.datatype PrimitiveList Buffer]
+           )
+  )
+
+(defn append-to-file [fpath bytes]
+  (with-open [o (io/output-stream fpath :append true)]
+    (.write o bytes)))
```

Comment:
The output stream could be a mutable variable on the list:
`^{:unsynchronized-mutable true ^tag OutputStream} output-stream`

Same idea with the mmap handle.

Reply:
for output stream we would gain 20%